src/plugins/irc-events/link.js

```diff
@@ -64,7 +64,7 @@ function parse(msg, url, res, client) {
 	case "image/gif":
 	case "image/jpg":
 	case "image/jpeg":
-		if (res.size < (Helper.config.prefetchMaxImageSize * 1024)) {
+		if (res.text.length < (Helper.config.prefetchMaxImageSize * 1024)) {
```

Comment:
Because the connection might timeout or drop, so our `text` can be partial (e.g. received 15kb but content-length is 300kb)